python/node_similarity.py

```diff
@@ -0,0 +1,285 @@
+from math import sqrt
+from typing import Set, List, Callable, Tuple, Union
+from itertools import product
+import mgp
+
+# not implemented for directed graphs
+# not implemented for multigraphs
+# not taking into account edge weights
+
+neighbors_dict = dict()
+
+
+class Mode:
+    """
+    Constants for the mode parameter
+    """
+
+    PAIRWISE = ["p", "pairwise"]
+    CARTESIAN = ["c", "cartesian"]
+
+
+@mgp.read_proc
+def jaccard(
+    context: mgp.ProcCtx, node1: mgp.Any, node2: mgp.Any, mode: str = "cartesian"
```

Comment:
Does it make sense that default is pairwise as it reduces complexity for calculation

Reply:
I don't think it matters. In the end, the user will decide how they want to use the function.